website/thaliawebsite/templates/base.html

```diff
@@ -108,6 +108,15 @@
                 {% endblocktrans %}
             </p>
         </div>
+    {% elif request.user and request.user.profile and not request.user.profile.photo %}
+        <div class="announcement">
+            <p>
+                {% blocktrans trimmed %}
+                You don't have a profile picture yet! Upload one on your
+                <a href="https://thalia.nu/user/edit-profile/">profile page</a> so that the other members know who you are. :)
```

Comment:
Instead of a hardcoded url, you can generate a url using the `url` template command. You can find the name of the page here: https://github.com/svthalia/concrexit/blob/d87ffccd25d8adb163bb6263a66985e98afdd266/website/members/urls.py#L34-L37

And more info on the url command here: https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#std-templatetag-url